downstream/modules/platform/proc-gw-editing-a-user.adoc

```diff
@@ -6,6 +6,15 @@
 
 You can modify the properties of a user account after it is created.
 
+In upgrade scenarios, there might be pre-existing user accounts from {ControllerName}, {EDAName} or {HubName} services. When editing these user accounts, the *User type* checkboxes indicate whether the account had service level administrator privileges. You can revoke or assign administrator permissions for the individual services and designate the user as either an {PlatformNameShort} Administrator, {PlatformNameShort} Auditor or Normal User. Assigning administrator privileges to all of the individual services automatically designates the user as an {PlatformNameShort} Administrator. See xref:proc-controller-creating-a-user[Creating a user] for more information about user types.
+
+To see whether a user had service level auditor privileges, you must refer to the API.
+
+[NOTE]
+====
+Users previously designated as automation controller or automation hub administrators are labeled as *Normal* in the *User type* column in the xref:proc-gw-users-list-view[Users list view]. You can see whether these users have administrator privileges, from the *Edit Users* page.
```

Comment:
I'd add attributes for controller and hub in this line.

Reply:
Fixed.